### PR TITLE
NPC Dialogue Update 0.2: Add several more talk snippets

### PR DESCRIPTION
**Pull request by I-am-Erk**
#### Summary

```SUMMARY: Content "NPC Dialogue - add more snippets"```

#### Purpose of change

Snippets are a useful way of replacing expletives and common phrases in dialogue, to add variation. With further NPC dialogue coming in, it is a good idea to expand these now so that they can be integrated from the start... it's a pain to go back and add snippets to old text.

#### Describe the solution

Mostly single word substitutions to avoid repeating the same terms over and over.

#### Describe alternatives you've considered

I may add more snippets before the day is done. They don't have to have a ton of synonyms to start with, as long as they're things we'll often need.

#### Additional context

This was previously part of another pull request that is being split up, https://github.com/CleverRaven/Cataclysm-DDA/pull/26834/. This does include some changes: I had split snippets into two files initially, and I could again I suppose, but in hindsight I'm not sure it adds anything.

What we might want is a comment at the top of the file listing available snippets.

#### Progress
- [x] add snippets that I have already used in dialogue files
- [x] Spell check / typo check snippets
- [x] Test implementation in-game

### CODE REVIEW

File: data/json/npcs/talk_tags.json [hunk 1]
```diff
@@ -537,5 +537,81 @@
...
+            "walking corpse", 
+            "<swear> <zombie>", 
+            "living dead" 
+        ]
+    },{
+        "type": "snippet",
+        "category": "<zombies>",
+        "text": [ 
+            "zombies", 
+            "Z",
```
Comment (anothersimulacrum) [inline data/json/npcs/talk_tags.json]: This should probably be "Zs", or else it sounds like it is just referring to one zombie.

File: data/json/npcs/talk_tags.json [hunk 2]
```diff
@@ -537,5 +537,81 @@
...
+            "living dead" 
+        ]
+    },{
+        "type": "snippet",
+        "category": "<zombies>",
+        "text": [ 
+            "zombies", 
+            "Z", 
+            "dead", 
+            "<swear> zombies",
```
Comment (anothersimulacrum) [inline data/json/npcs/talk_tags.json]: Do you mean <swear> <zombies>?

File: data/json/npcs/talk_tags.json (hunk 1 above)
Comment (I-am-Erk) [reply data/json/npcs/talk_tags.json]: Both work, I think, and that could be added. 

"I ran into a bunch of Z back there"
"I can't believe how many Z's we saw"

File: data/json/npcs/talk_tags.json (hunk 2 above)
Comment (I-am-Erk) [reply data/json/npcs/talk_tags.json]: Didn't mean it, but no reason for it to not be that.

Comment (anothersimulacrum) [reply data/json/npcs/talk_tags.json]: In zombies there is &lt;swear&gt; &lt;zombie&gt;, so I figured it was following that pattern and you just forgot to add the &lt;&gt;.

Comment (I-am-Erk) [reply data/json/npcs/talk_tags.json]: I think at the time, I was thinking that I'd like 'zombies' to come up more often, but that was when I thought I'd have more esoteric weird terms in the snippet. Most of these are terms people would use often, so putting the second term as a snippet works just fine

**Pull request merged**